config/locales/en.yml

```diff
@@ -431,9 +431,7 @@ en:
         matching_details:
           title: "Is this claim still valid despite having matching details with other claims?"
         payroll_details:
-          title:
-            "The claimant’s %{bank_or_building_society} details have not been automatically validated. Has the claimant
-            confirmed their %{bank_or_building_society} details?"
+          title: "%{claimant_name}'s %{bank_or_building_society} details have not been automatically validated. Has %{claimant_name} confirmed their %{bank_or_building_society} details?"
```

Comment:
could all of these for `student_loans`, `additional_payments` and `further_education_payments` be added to a single top-level `admin.task_questions.payroll_details.title`, as they are all the same now?